extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 class ARecoveryModule {
+	const disabledMessage = PHP_EOL . '<!-- Recovery disabled. -->' . PHP_EOL;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant "disabledMessage" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)